common/src/node.rs

```diff
@@ -39,6 +42,158 @@ impl Graph {
             (math::mtranspose(&position.local) * node.state.up_direction()).xyz(),
         ))
     }
+
+    pub fn get_chunk_neighbor(
```

Comment:
Due to the merging of `DualGraph` and `Graph`, all these new member functions should probably move to `graph.rs`. Having two `impl Graph`s is a bit confusing.

I was hesitant to do that because they would have been in `DualGraph` before the merge, but I don't think there's a clear boundary between what should live in what file, so consistency is key.

- [ ] TODO: Combine `impl Graph` blocks.